app-backend/db/src/test/scala/com/azavea/rf/database/UserGroupRoleDaoSpec.scala

```diff
@@ -57,6 +58,35 @@ class UserGroupRoleDaoSpec extends FunSuite with Matchers with Checkers with DBT
     }
   }
 
+  test("list users by group") {
+    check {
+      forAll {
+        (userCreate: User.Create, manager: User.Create, orgCreate: Organization.Create, teamCreate: Team.Create,
+         platformCreate: Platform.Create, ugrCreate: UserGroupRole.Create, ugrUpdate: UserGroupRole.Create, page: PageRequest) => {
+          val insertUgrWithRelationsIO = for {
+            orgAndUser <- insertUserAndOrg(userCreate, orgCreate)
+            (insertedOrg, insertedUser) = orgAndUser
+            managerUser <- UserDao.create(manager.copy(organizationId = insertedOrg.id))
+            teamAndPlatform <- (
+              TeamDao.create(teamCreate.copy(organizationId = insertedOrg.id).toTeam(managerUser)),
+              PlatformDao.create(platformCreate.toPlatform(managerUser))
```

Comment:
As mentioned above, I'd change`platformCreate` to `platform` I think we could just do `PlatformDao.create(platformCreate)` now since the latest changes in the anchor PR.